package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangolindex/components",
-  "version": "3.0.0-rc.1",
+  "version": "3.0.0",
```

Comment:
we dont need to update version here, it will be automatically updated when we merge it